pkg/controller/jobs/deployment/deployment_webhook.go

```diff
@@ -81,12 +82,13 @@ func (wh *Webhook) Default(ctx context.Context, obj runtime.Object) error {
 		if deployment.Spec.Template.Labels == nil {
 			deployment.Spec.Template.Labels = make(map[string]string, 1)
 		}
+		deployment.Spec.Template.Labels[constants.ManagedByKueueLabelKey] = constants.ManagedByKueueLabelValue
 		queueName := jobframework.QueueNameForObject(deployment.Object())
 		if queueName != "" {
-			deployment.Spec.Template.Labels[constants.QueueLabel] = queueName
+			deployment.Spec.Template.Labels[controllerconstants.QueueLabel] = queueName
```

Comment:
Please revert drive-by cleanups. The PR is already large.

Reply:
Than not sure how to name `sigs.k8s.io/kueue/pkg/constants` package.

https://github.com/kubernetes-sigs/kueue/blob/45c67c81a3dd21a86c84dc7384556658bdd72caf/pkg/controller/jobs/deployment/deployment_webhook.go#L31-L32

`pkgconstants` sound weird.

Reply:
The change itself might be ok, but it just does not belong to the PR, we can have a preparatory PR, but again, I'm not sure why we need any changes to the labels of deployment

Reply:
Ack, after I understand why we change Deployment I'm ok with that change.